telegram/start_telegram_bot.js

```diff
@@ -480,17 +480,25 @@ module.exports = (args, cbk) => {
           return cbk();
         }
 
-        inquirer.prompt([interaction.user_id_prompt]).then(({code}) => {
-          if (!code) {
-            return cbk([400, 'ExpectedNumericConnectCode']);
-          }
-
-          connectedId = code;
-
-          return cbk();
-        });
+          return ask({
+            type: 'input',
+            message: interaction.user_id_prompt.message,
+            name: interaction.user_id_prompt.name,
+            validate: code => {
+              if (!code || !isNumber(code)) {
+                return `Expected Numeric Connect Code`;
+              }
 
-        return;
+              if (code === args.key) {
```

Comment:
it should be key starts with code because the bot api key is like botid:password not just botid and sometimes people enter in the botid